App/components/Image/image.styled.ts

```diff
@@ -1,16 +1,14 @@
 import styled from 'styled-components/native'
+import { BorderType } from '../Box/box.types'
 
 type ImageContainerProps = {
-  image: string
   width?: number
   height?: number
-  border?: string
   borderRadius?: number
 }
```

Comment:
Shouldn't be there all props (where is image source prop?)

Reply:
in image.tsx. Set as source={{ uri: image }} so we dont need it here in styled file

Reply:
border is also set in image.tsx but the image is wrapped into box